engine/src/main/java/org/hibernate/search/bridge/IgnoreAnalyzerBridge.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.bridge;
+
+
+/**
+ * This interface is used to mark the bridges we want to keep safe from the analyzers.
+ *
+ * @author Guillaume Smet
+ */
+public interface IgnoreAnalyzerBridge {
```

Comment:
Should this be a public contract right away? If so, should it be marked as experimental? I'd expect this to be morphed into a method on `FieldBridge` a la `boolean applyAnalyzers()` down the road instead of it being a marker interface?